packages/runtime/datastore/package.json

```diff
@@ -128,7 +128,6 @@
 		"@fluidframework/runtime-definitions": "workspace:~",
 		"@fluidframework/runtime-utils": "workspace:~",
 		"@fluidframework/telemetry-utils": "workspace:~",
-		"lodash": "^4.17.21",
```

Comment:
Don't you need to add the @ungap/structured-clone dependency?

Reply:
CI seems to be passing without it. Any thoughts @zhenmichael or @anthony-murphy?

Reply:
I guess structuredClone is a built-in for browsers for the last couple of years. I wasn't aware.

Reply:
Sounds like loop has had issues with structuredClone, so I included the @ungap dependency in the latest commit